scripts/ci

```diff
@@ -135,13 +135,13 @@ test_failed()
 # Background to start the Docker pulls
 ./test-warm >/dev/null &
 
-CI=true MAVEN_ARGS='-B -q' run ./build
-
 threads=$(($(nproc) + 1))
 if [ $threads -gt 8 ]; then
     threads=8
 fi
 
+CI=true MAVEN_ARGS="-B -q -T $threads" run ./build
```

Comment:
This isn't stable.  I've tried it in the past and sometimes maven just randomly fails.  Maybe it's better in newer versions of maven?  I tried with one in ubuntu:14.04